src/components/ReleaseNotesVersions/ReleaseNotesVersions.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ReleaseNotesVersions from "./ReleaseNotesVersions";
+import { isExternalDomain } from "./ReleaseNotesVersions";
+import { useHistory } from "@docusaurus/router";
+
+// Mock any external modules or hooks here
```

Comment:
I reckon remove this whole test file. 😭

Reply:
I'm gonna keep it just to catch component signature changes.